cypress/support/hostsfunctions.ts

```diff
@@ -147,6 +148,7 @@ export function checkHostGroup(host_type: string, organization: Organization) {
       cy.clickModalButton('Confirm');
       cy.contains('button', 'Close').click();
```

Comment:
please user `cy.getByDataCy` command for this
same for line 150 and 151

Reply:
thanks 
will do
thanks!

Reply:
that's not working for me at the moment 
why is that change needed?

Reply:
We will fix this later, sorry, I need to merge this PR so I can also work on that ASAP.